pkg/datastore/postgres.go

```diff
@@ -15,72 +14,45 @@ import (
 	"github.com/jmoiron/sqlx"
 	v1 "github.com/metal-stack/masterdata-api/api/v1"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 
 	// import for sqlx to use postgres driver
 	_ "github.com/lib/pq"
 )
 
-// exchangable for testing
+// exchangeable for testing
 var Now = time.Now
```

Comment:
Also not related but this should not be public. I think a nicer way is to patch the function only in unit test files, such that production code is not altered at all: https://github.com/metal-stack/metalctl/blob/v0.12.8/cmd/common_test.go#L28-L32

Reply:
Will do in the the next PR